src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -426,14 +426,14 @@ void verifyPassportAuth(UUID snapshotId, DRSPassportRequestModel drsPassportRequ
   }
 
   private DRSObject lookupDRSObjectAfterAuth(
-      boolean expand,
+      Boolean expand,
       SnapshotCacheResult snapshot,
       DrsId drsId,
       AuthenticatedUserRequest authUser,
       boolean passportAuth,
       String billingSnapshot) {
     SnapshotProject snapshotProject = getSnapshotProject(snapshot.id);
-    int depth = (expand ? -1 : 1);
+    int depth = (Optional.ofNullable(expand).orElse(false) ? -1 : 1);
```

Comment:
This is OK but I think it's better to perform defaulting at the outermost possible level, and make this parameter `boolean` everywhere else. It looks like there are two outer sources of this value, `DataRepositoryServiceApiController.getObject()` and `DrsService.lookupObjectByDrsIdPassport()`. It looks like the `getObject()` parameter is defaulted via an annotation so that shouldn't be a source of `null`, so the only case that needs a check is in `lookUpObjectByDrsIdPassport()`.

Reply:
As discussed, I changed approach to define the default in our OpenAPI spec.